### PR TITLE
Gitcoin is a suite of OSS incentivization tools.  This PR makes the new product page front and center

### PR DESCRIPTION
**Pull request by owocki**
Given that gitcoin is no longer just a bounty platform, and to clarify the brand navigation, i believe it would behoove us to make the landing page at gitcoin.co/ representative of our suite of products and how they fit together.

This PR makes the new product page ( https://github.com/gitcoinco/web/issues/3278#issue-393685127 ) front and center, (it will load on https://gitcoin.co/)  For targeted campaigns, we can and should still still send users to `/funder` or `/contributor`.

If we achieve consensus that this should be merged, I think we should also
1. Put all of the current brochureware pages (products, mission, about, etc -- pictured [here](https://github.com/gitcoinco/web/issues/2867#issuecomment-449581486)) into a mini nav as specified in [this design](https://github.com/gitcoinco/web/issues/2867#issuecomment-449581486)
1. update the nav on all of the brochureware pages to be a brochureware-specific nav (just like we have a kudos nav, grants nav, etc)
1. namespace all of the brochureware pages to `/about/mission`, `/about/not_a_token`, etc
1. namespace all of the bounty-specific stuff to `/bounties/explorer`, `/bounties/spacemeshos/app/21/1556`, etc
1. whatever updates we make, make sure we make the same updates on codefund.

Doing the above would make it clear to the user that Gitcoin is a parent brand, and that Kudos, Grants, CodeFund, Bounties, are all sub-brands.

### CODE REVIEW

File: app/retail/templates/shared/top_nav.html [hunk 1]
```diff
@@ -18,7 +18,7 @@
 {% load i18n static matches %}
 <div class="top-nav g-font-muli {{class}}">
   <a class="link d-md-block d-none" href="{% url 'products' %}">Grow Open Source</a>
-  <a class="{% if request.path == '/' %}selected{% endif %}" href="/">
+  <a class="{% if request.path == '/' %}selected{% endif %}" href="{% url 'funder' %}">
```
Comment (mbeacom) [inline app/retail/templates/shared/top_nav.html]: Should we keep these consistent? 🤔

Comment (owocki) [reply app/retail/templates/shared/top_nav.html]: good catch.  will update IFF @PixelantDesign is okay with the direction i'm taking this.

**Comment by vs77bb** [comment]
I think we need to add this to the Blue Sprint board. Thoughts, @PixelantDesign @frankchen07?

**Comment by frankchen07** [comment]
I think this might belong on Green Squad - I was poking around to see if there was any type of hypothesis testing. There might be some reusable stuff that we can re-purpose for the "that was easy" value proposition.

**Comment by thelostone-mc** [comment]
Closing this out as we've got the name homepage coming in and that would address this

**Pull request closed without merging**